Userland/Libraries/LibBencode/Bencode.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2021, Cesar Torres <shortanemoia@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Bencode.h"
+#include <AK/Format.h>
+#include <AK/Optional.h>
+#include <AK/String.h>
+
+namespace Bencode {
+
+void BencodeEncoder::append(i64 value)
+{
+    m_builder.append(String::formatted("i{}e", value));
```

Comment:
```suggestion
    m_builder.appendff("i{}e", value);
```
(Same below)

Reply:
Good point

Reply:
Hmm, if it does its a bug in AK, so you might want to open an issue for that

Reply:
Yeah disregard that, it was a type on my end. It works now :)